src/renderer/d3d.rs

```diff
@@ -401,7 +404,7 @@ impl SkiaRenderer for D3DSkiaRenderer {
             self.gr_context.submit(Some(SyncCpu::No));
 
             tracy_gpu_zone!("present");
-            if self.swap_chain.Present(1, 0).is_ok() {
+            if self.swap_chain.Present(1, DXGI_PRESENT::default()).is_ok() {
```

Comment:
I would prefer if we are specific here with 0, since the default is not explicitly documented to mean something and could therefore change. `0` has a documented meaning here https://learn.microsoft.com/en-us/windows/win32/direct3ddxgi/dxgi-present